dbms/tests/queries/0_stateless/00562_in_subquery_merge_tree.sql

```diff
@@ -0,0 +1,28 @@
+DROP TABLE IF EXISTS test.merge_tree_in_subqueries;
```

Comment:
I have not included any tests for NULL values since there are a few issues with them not related to the current patch.

For example, the query below prints 'Cannot get value from Nothing'.

SELECT *
FROM t
WHERE id IN (SELECT NULL)